Request/ParamConverter/DateTimeParamConverter.php

```diff
@@ -46,6 +46,13 @@ public function apply(Request $request, ParamConverter $configuration)
         if (isset($options['format'])) {
             $date = DateTime::createFromFormat($options['format'], $value);
 
+            if (isset($options['strict']) && true === $options['strict']) {
+                $errors = DateTime::getLastErrors();
+                if (0 < $errors['warning_count']) {
+                    $date = false;
+                }
+            }
+
```

Comment:
This whole part could be refactored into the if statement below.

```
isset($options['strict']) && true === $options['strict'] && 0 < DateTime::getLastErrors()['warning_count']
```

Works since 5.4: https://secure.php.net/manual/en/language.types.array.php#example-62